tests/kernel/iop/messagebox/mbx-common.c

```diff
@@ -0,0 +1,32 @@
+#include <thbase.h>
+#include <thmsgbx.h>
+#include "mbx-common.h"
+
+u32 getThreadPriority(s32 threadId) {
+	iop_thread_info_t threadInfo = {};
+	ReferThreadStatus(threadId, &threadInfo);
+	return threadInfo.currentPriority;
+}
+
+void printMbx(s32 mbxId) {
+	iop_mbx_status_t mbxInfo;
+	s32 result = ReferMbxStatus(mbxId, &mbxInfo);
```

Comment:
Oh sorry, I completely missed this! No idea if the reserved values contain anything interesting, but I'll take a look. And yeah, attr and option values should definitively be logged.